.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+install:
+  - make requirements-test
+  - make -C test pre-populate-cache
+script:
+  - make -C test
```

Comment:
So this line just runs everything on the base/ root level of the test/ makefile?

Reply:
It runs the default goal which is currently `test-job-templates`

Reply:
I've changed this line to be `make test` which is more self-descriptive